File: prg.py
# for i in range(500):
#     if(1.75*i==700):
#         print("There should be ₹",i," coins")

#There are 60eyes and 86 foot in total find total no of goats and hens
# for i in range(1,90):
#     for j in range(1,60):
#         if (4*i)+(2*j)==86 and (i+j)*2==60:
#             print("There are ",i,"goats and ",j," hens")

# for i in range(1,102,2):
#     for j in range(101,0,-3):
#         if(i==j):
#             print(i," is the common")

ins = 0
bac = 13312
day=10
while day>1:
    ins = bac/2
    bac = ins
    day -=1
print(ins)

File: prg.java
// import java.io.*;
// class prg
// {
//     public static void main(String[] args)
//     {
//         for(int i=1;i<=500;i++)
//         {
//             if(0.25*i+0.5*i+1*i==700)
//             {
//                 System.out.println("There should be ₹"+i+" coins");
//             }
//         }
//     }
// }

// class prg
// {
//     public static void main(String[] args) {
//         int day=10,bac=13312,ins=0;
//         while(day>1)
//         {
//             ins = bac/2;
//             bac = ins;
//             day--;
//         }
//         System.out.println(ins);
//     }
// }

import java.util.*;
class prg
{
    public static void main(String[] args) 
    {
        int i,sum=0,ch;
        Scanner sc = new Scanner(System.in);
        do
        {
            System.out.println("Enter a number");
            i = sc.nextInt();
            sum += i;
            System.out.println("Do you want to still continue 1 for yes 0 for no");
            ch = sc.nextInt();
        }while(ch!=0);
        System.out.println(sum);
        sc.close();
    }
}